weave/trace_server/calls_query_builder.py

```diff
@@ -656,6 +660,11 @@ def _as_sql_base_format(
             pb, table_alias, self.query_conditions
         )
 
+        # Add datetime optimization based on UUIDv7 timestamp
```

Comment:
Given that this is a short-term hack - I would recommend clearly naming and articulating that this is a work around and document the limitations and assumptions